### ISSUE TITLE
Mark-I: OpenSSL deprecation warning when using mycroft-msm

### ISSUE THREAD

**Issue by domcross** [MycroftAI/mycroft-core#2204]
* I'm running a Mark 1
* With version 19.2.13 of the Mycroft software

1. install a new skill: `mycroft-msm install ...`
2. during installation of depencies there is a deprecation warning about OpenSSL issued:
```
/opt/venvs/mycroft-core/lib/python3.4/site-packages/cryptography/hazmat/bindings/openssl/binding.py:163: 
CryptographyDeprecationWarning: OpenSSL version 1.0.1 is no longer supported by the
OpenSSL project, please upgrade. A future version of cryptography will drop support for it.
  utils.CryptographyDeprecationWarning
```

**Comment by forslund** [comment]
Thanks for reporting this. I've seen it and we're working to fix it for the next release.

**Comment by domcross** [comment]
fixed with #2253 - 19.8.3
